src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsstafflookupservice/integration/search/SearchStaffByEmail.kt

```diff
@@ -0,0 +1,67 @@
+package uk.gov.justice.digital.hmpps.hmppsstafflookupservice.integration.search
+
+import kotlinx.coroutines.runBlocking
+import org.junit.jupiter.api.Test
+import uk.gov.justice.digital.hmpps.hmppsstafflookupservice.db.entities.Staff
+import uk.gov.justice.digital.hmpps.hmppsstafflookupservice.integration.IntegrationTestBase
+
+class SearchStaffByEmail : IntegrationTestBase() {
+
+  @Test
+  fun `must return staff by partial matched email`(): Unit = runBlocking {
```

Comment:
These tests are very anaemic but not sure how far we should go.

Personally, I think it would be better to have one or more sets of data to test against and have parameterised tests against each.

We can sort that later

Reply:
No need - we are effectively testing PostgreSQL here so there is no real need to have lots of tests.

We can maybe add them if we get lots of questions about how it works.